src/pytest_allure_spec_coverage/matcher.py

```diff
@@ -368,6 +370,9 @@ def pytest_terminal_summary(self, terminalreporter: TerminalReporter):
     def pytest_sessionfinish(self, session, exitstatus):
         """After terminal summary we print spec coverage success message"""
         yield
-        if self.config.fail_under and exitstatus != ExitCode.NO_TESTS_COLLECTED:
+        if (self.config.fail_under and exitstatus != ExitCode.NO_TESTS_COLLECTED) or self.nonexistent:
             terminal = session.config.pluginmanager.get_plugin("terminalreporter")
             terminal.write_line(f"Spec coverage is greater than target {self.config.fail_under}%! 🎉🎉🎉")
+            if self.nonexistent:
+                tests_without_spec = ", ".join([item.name for item in self.nonexistent])
+                terminal.write_line(f"There are tests without spec: {tests_without_spec}")
```

Comment:
If there are a lot of such tests output will be messy. Consider adding list formatting